qless/workers/forking.py

```diff
@@ -82,6 +82,8 @@ def run(self):
                     os.chdir(sandbox)
                     try:
                         self.spawn(resume=resume[index], sandbox=sandbox).run()
+                    except:
+                        logger.exception('Exception in spawned worker')
```

Comment:
No trace? If you're worried about spammy logs, how about at least the exception type?